pkg/cli/admin/inspect/inspect.go

```diff
@@ -113,6 +114,7 @@ func NewCmdInspect(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd.Flags().StringVar(&o.DestDir, "dest-dir", o.DestDir, "Root directory used for storing all gathered cluster operator data. Defaults to $(PWD)/inspect.local.<rand>")
 	cmd.Flags().StringVar(&o.eventFile, "events-file", o.eventFile, "A path to an events.json file to create a HTML page from")
 	cmd.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", o.allNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
+	cmd.Flags().BoolVar(&o.podLogFiles, "pod-log-files", o.podLogFiles, "If present, retrieve all log files(including rotated logs) that are available for selected pods.")
```

Comment:
How this flag plays with the next two? I don't see any interaction, but that will be confusing to a lot of users. You should properly wire both `--since` and `--since-time` in this logic, especially that the pod logs can be big at times.

Reply:
Make it explicit in the flag description that this is best-effort and doesn't guarantee any value.

Reply:
Supporting `--since/--since-time` would require implementing what is there in kubelet and possibly extracting multiple archives.
I've hidden the option as discussed offline and provided a better help message.